overlord/snapstate/autorefresh_gating.go

```diff
@@ -272,6 +322,36 @@ func ProceedWithRefresh(st *state.State, gatingSnap string) error {
 	return nil
 }
 
+// ResumeRefreshes removes the gating information on the heldSnaps by the gatingSnap.
+func ResumeRefreshes(st *state.State, gatingSnap string, heldSnaps []string) error {
```

Comment:
would it be worth unifying somehow this and ProceedWithRefresh, with a way to indicate all heldSnaps?